ruby/README.md

```diff
@@ -19,7 +19,7 @@
 
 # Apache Arrow Ruby
 
-There are the official Ruby bindings for Apache Arrow.
+These are the official Ruby bindings for Apache Arrow.
```

Comment:
Good catch!
BTW, what do you think about "Here are" instead?

```suggestion
Here are the official Ruby bindings for Apache Arrow.
```

Reply:
Hi,
that could also work. Might even make more sense since using "These" might imply the bindings to be in list format. So "Here are" sounds good! Nice improvement.